### DISCUSSION TITLE
Pop up is not shown in MAC

### DISCUSSION THREAD

**Discussion by juliussaputra** [SafeExamBrowser/seb-mac#163] [Q&A] [answered]
am currently testing the SEB for our basic programming exams.
For the exam we are using a third party IDE ([pythonandturtle.com](https://pythonandturtle.com/)).

Problem: The input prompt window from the IDE is not shown in SEB MAC(**3.0**) but in windows (SEB **3.3.1**) it is working without any problem.
Both case were tested with the same SEB config file on MAC OS Monterey 12.0.1 .

Screenshots from both cases are attached:
[SEB_Popup_Windows](https://user-images.githubusercontent.com/17081818/144246947-74d86130-e8cb-4496-b9d6-8c46c0bc67a9.jpeg)
[SEB_Popup_Mac](https://user-images.githubusercontent.com/17081818/144246959-15f9e529-3808-4839-95db-70c66a0a93ec.jpeg)

For the test, I have tried SEB with the w3schools link as well: https://www.w3schools.com/js/tryit.asp?filename=tryjs_prompt. The result was the same, the prompt window is just not shown and return empty string value as a result.

is there any specific setting for SEB in MAC that allows the input window? or is there any workaround for this case?

Many thanks in advance for your work!
Julius

**Comment by danschlet** [comment, marked as answer]
You're right, the Javascript text input panel is not implemented yet in SEB 3.0. We'll fix it in an update.

**Reply by juliussaputra** [reply]
Thank you! will look forward for the update!
